benchmarks/helics/echoMessageBenchmarks.cpp

```diff
@@ -8,182 +8,180 @@ SPDX-License-Identifier: BSD-3-Clause
 #include "EchoMessage.hpp"
 #include "helics/core/BrokerFactory.hpp"
 #include "helics/core/CoreFactory.hpp"
+#include "helics/helics-config.h"
 #include "helics_benchmark_main.h"
-#include <benchmark/benchmark.h>
 
+#include <benchmark/benchmark.h>
 #include <chrono>
 #include <fstream>
+#include <gmlc/concurrency/Barrier.hpp>
 #include <iostream>
 #include <thread>
 
-#include <gmlc/concurrency/Barrier.hpp>
-
-#include "helics/helics-config.h"
-
 // static constexpr helics::Time tend = 3600.0_t;  // simulation end time
 using namespace helics;
 
-static void BM_echo_singleCore (benchmark::State &state)
+static void BM_echo_singleCore(benchmark::State& state)
 {
     for (auto _ : state)
     {
-        state.PauseTiming ();
+        state.PauseTiming();
 
-        int feds = static_cast<int> (state.range (0));
-        gmlc::concurrency::Barrier brr (static_cast<size_t> (feds) + 1);
-        auto wcore = helics::CoreFactory::create (core_type::INPROC, std::string ("--autobroker --federates=") +
-                                                                       std::to_string (feds + 1));
+        int feds = static_cast<int>(state.range(0));
+        gmlc::concurrency::Barrier brr(static_cast<size_t>(feds) + 1);
+        auto wcore = helics::CoreFactory::create(
+            core_type::INPROC, std::string("--autobroker --federates=") + std::to_string(feds + 1));
```

Comment:
Can we make it such that the closing bracket is one a new line? i.e.:

```cpp
        auto wcore = helics::CoreFactory::create(
            core_type::INPROC, std::string("--autobroker --federates=") + std::to_string(feds + 1)
        );
```

Reply:
Do you know if there is an option for that?  I didn't see anything obvious that controlled that

Reply:
There's a patch for it but it hasn't made it in yet.

https://reviews.llvm.org/D33029

So I guess this is not possible right now.